bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -142,6 +142,7 @@ def aws_stages
 
     def google_stages
       [
+        :rsyslog_config,
```

Comment:
This stage must be run for the Google stemcell to provide a correct `/etc/init/rsyslog.conf`